Assets/MixedRealityToolkit.Tests/PlayModeTests/PressableButtonTests.cs

```diff
@@ -125,6 +125,50 @@ public IEnumerator PressButtonWithHand()
             yield return null;
         }
 
+        /// <summary>
+        /// This test reproduces P0 issue 4566 which didn't trigger a button with enabled backpressprotection 
+        /// if hands were moving too fast in low framerate
+        /// </summary>
+        [UnityTest]
+        public IEnumerator PressButtonFast()
+        {
+            GameObject testButton = InstantiateSceneAndDefaultPressableButton();
+
+            // Move the camera to origin looking at +z to more easily see the button.
+            MixedRealityPlayspace.PerformTransformation(
```

Comment:
I think Lars has a convenience method in his latest interactable tests pr that you can call instead of doing manually.